code/game/area/Space Station 13 areas.dm

```diff
@@ -49,6 +49,8 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	var/turf/base_turf //The base turf type of the area, which can be used to override the z-level's base turf
 	var/vessel = "NEV Northern Light" //The ship or station this area is on. This is so far just for the benefit of shield generators
 	var/holomap_color // Color of this area on station holomap
+	var/narrate //!!MODULAR OCCULUS ADDITION!! A text-based description of what this area is for.
```

Comment:
This isn't modular. It can be made modular however.

Reply:
> This isn't modular. It can be made modular however.

iiiiit didn't work when i made it modular. i tried to make the majority of this modular where i could.